src/CSFML/Network/IpAddress.cpp

```diff
@@ -48,7 +48,7 @@ namespace
 
 
 ////////////////////////////////////////////////////////////
-const sfIpAddress sfIpAddress_None = sfIpAddress_fromBytes(0, 0, 0, 0);
+const sfIpAddress sfIpAddress_None = {{0}};
```

Comment:
Is this not identical to what we previously had? I believe `sfIpAddress_fromBytes(0, 0, 0, 0)` should produce an array of all zeros.

Reply:
It would produce the "0.0.0.0" address, but this isn't what `sfIpAddress_None` should be. It should represent the same state as `std::nullopt` for `std::optional<sf::IpAddress>`, i.e. an invalid / missing IP address. Initializing the char array with zeroes should make it act as an empty string, which would make it distinct from `sfIpAddress_Any`.